lib/ui/tutorials/tutorial_sliders.dart

```diff
@@ -0,0 +1,286 @@
+import 'package:flutter/services.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:intro_slider/intro_slider.dart';
+import 'package:intro_slider/slide_object.dart';
+
+import '../../ui_imports.dart';
+
+class TutorialSliders extends StatefulWidget {
+  @override
+  _TutorialSlidersState createState() => _TutorialSlidersState();
+}
+
+class _TutorialSlidersState extends State<TutorialSliders> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  void onDonePress() {
+    // Do what you want
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    List<Slide> slides = new List();
+    slides.add(new Slide(
+      title: "Disclaimer",
+      description:
```

Comment:
localized strings for all but maybe we can create a small tutorial and then discuss it with helmi?